ide/coqide/wg_Find.ml

```diff
@@ -136,14 +138,17 @@ class finder name (view : GText.view) =
         view#buffer#end_user_action ()
       end
 
-    method private set_not_found () =
-      find_entry#misc#modify_bg [`NORMAL, `NAME "#F7E6E6"];
+    method private set_not_found =
```

Comment:
I think that it is a conscious convention to have methods of type `unit -> unit` to signal they are effectful.

Reply:
OK, thanks for the info. I reverted my change.